src/rtk/features/apiDelegation/delegationRequest/delegationRequestSlice.ts

```diff
@@ -87,22 +87,31 @@ const delegationRequestSlice = createSlice({
   extraReducers: (builder) => {
     builder
       .addCase(postApiDelegation.fulfilled, (state, action) => {
+        const sucessfulDelegations = state.succesfulApiDelegations;
         const delegation: ApiDelegation = {
           apiName: action.payload.apiName,
           orgName: action.payload.orgName,
         };
-        state.succesfulApiDelegations.push(delegation);
+
+        sucessfulDelegations.push(delegation);
+        state.succesfulApiDelegations = sucessfulDelegations.sort((a, b) =>
+          a.orgName.localeCompare(b.orgName),
+        );
         state.batchPostCounter += 1;
         if (state.batchPostCounter === state.batchPostSize) {
           state.loading = false;
         }
       })
       .addCase(postApiDelegation.rejected, (state, action) => {
+        const failedDelegations = state.failedApiDelegations;
         const delegation: ApiDelegation = {
           apiName: action.meta.arg.apiName,
           orgName: action.meta.arg.orgName,
         };
-        state.failedApiDelegations.push(delegation);
+        failedDelegations.push(delegation);
+        state.failedApiDelegations = failedDelegations.sort((a, b) =>
+          a.orgName.localeCompare(b.orgName),
```

Comment:
Burde ikke teksten til høyre også sorteres alfabetisk?

<img width="1001" alt="image" src="https://github.com/Altinn/altinn-access-management-frontend/assets/31471142/5eb584a0-0cb8-4b39-ab48-4d05906ad4ff">

Reply:
Vel, hovedgrunnen til alfabetiseringen er at det da vil sortere alle delegeringene slik at de med samme mottaker står samlet.
Dersom vi skal også sortere for API så må nok sorteringsfunksjonen bli mer sofistikert, men det bør jo la seg gjøre ja

Reply:
Kom på en kortvint måte å gjøre det på så da gjorde jeg det sånn 😊